armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -463,34 +467,27 @@ def buReducingAssemblyRotation(self):
         hist = self.o.getInterface("history")
         for aPrev in self.moved:  # much more convenient to loop through aPrev first
             aNow = self.r.core.getAssemblyWithStringLocation(aPrev.lastLocationLabel)
-            if (
-                aNow in hist.getDetailAssemblies()
-            ):  # no point in rotation if there's no pin detail
+            # no point in rotation if there's no pin detail
+            if aNow in hist.getDetailAssemblies():
 
                 rot = self.getOptimalAssemblyOrientation(aNow, aPrev)
                 aNow.rotatePins(rot)  # rot = integer between 0 and 5
                 numRotated += 1
                 # Print out rotation operation (mainly for testing)
-                (
-                    i,
-                    j,
-                ) = aNow.spatialLocator.getRingPos()  # hex indices (i,j) = (ring,pos)
+                # hex indices (i,j) = (ring,pos)
+                (i, j) = aNow.spatialLocator.getRingPos()
```

Comment:
<- wack

Reply:
hey don't look to hard at this truly pre-historic part of the code.